src/java.base/share/classes/sun/net/www/MimeTable.java

```diff
@@ -198,42 +183,13 @@ public synchronized MimeEntry findByExt(String fileExtension) {
         return extensionMap.get(fileExtension);
     }
 
-    public synchronized MimeEntry findByDescription(String description) {
-        Enumeration<MimeEntry> e = elements();
-        while (e.hasMoreElements()) {
-            MimeEntry entry = e.nextElement();
-            if (description.equals(entry.getDescription())) {
-                return entry;
-            }
-        }
-
-        // We failed, now try treating description as type
-        return find(description);
-    }
-
-    String getTempFileTemplate() {
-        return tempFileTemplate;
-    }
-
     public synchronized Enumeration<MimeEntry> elements() {
```

Comment:
Can we drop `elements` as well?

Reply:
`sun.net.www.MimeTable#find` seems unused too

Reply:
In 82f2b3a14171d5963e2bfecb1b84105bda6a34f2, removed all unused methods and decreased the visibility (i.e., added `private`, and, if possible, `final`) wherever possible. (Started a `tier1` run in the meantime.)